### PR TITLE
ardupilotmega.xml: add MAV_CMD_DO_SEND_SCRIPT_MESSAGE

### PR DESCRIPTION
**Pull request by IamPete1**
Adds a command for use in missions to trigger scripting actions

We could potentially use all 7 params, I didn't want to confuse matters with the different types.

Or we could include a location

### CODE REVIEW

File: message_definitions/v1.0/ardupilotmega.xml [hunk 1]
```diff
@@ -48,6 +48,16 @@
         <param index="6">Empty.</param>
         <param index="7">Empty.</param>
       </entry>
+      <entry value="217" name="MAV_CMD_DO_SCRIPTING" hasLocation="false" isDestination="false">
```
Comment (peterbarker) [inline message_definitions/v1.0/ardupilotmega.xml:51]: Perhaps 
```suggestion
      <entry value="217" name="MAV_CMD_DO_SEND_SCRIPT_MESSAGE" hasLocation="false" isDestination="false">
```

It bothers me this is broadcast.  But I guess we've no way to distinguish scripts.

**Comment by IamPete1** [comment]
Because we only get 12 bytes in AP mission storage we can only do 3 floats.

**Comment by IamPete1** [comment]
https://github.com/ArduPilot/ardupilot/pull/16746

**Review by amilcarlucas — CHANGES_REQUESTED** [review]
It is probably better to move this above the 42000 value

**Comment by IamPete1** [comment]
@amilcarlucas if we go above 256 we can only store 10 bytes, so we could then only do two floats. We could do some tricks to store the third float as two `uint16_t`'s but that makes is all more complicated. 

Although it might be worth adding 10 of these in which case we might need to go above 256 in any-case.

**Comment by amilcarlucas** [comment]
At some point we will run out of < 256 MAV_CMDs

**Comment by IamPete1** [comment]
Turns out we can store a extra int16, so this is now a int16 ID and 3 floats

**Review by tridge — APPROVED** [review]

**Pull request merged**